src/pages/documentation/adobe-journey-optimizer-decisioning/release-notes.md

```diff
@@ -1,5 +1,14 @@
 # Release Notes
 
+## June 2, 2023
+
+### iOS Optimize 4.0.0
+
+Major version update for [Adobe Journey Optimizer - Decisioning](./index.md) for Adobe Experience Platform Mobile SDKs on iOS compatible with Mobile Core 4.0.0. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 11.0.
+* Include XCFrameworks built with Xcode 14.1 with the GitHub release.
+
```

Comment:
* Added support for using the Optimize SDK extension in [App Extensions](https://developer.apple.com/app-extensions/)